src/main/java/mate/academy/springboot/swagger/dto/RequestDto.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.springboot.swagger.dto;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
```

Comment:
Let's not use `@Data` annotation. It's better to use Getter Setter etc. Please fix the same issue in all places